src/battleSystem/battleEntity/Entity.py

```diff
@@ -9,7 +9,7 @@
 
 
 class Entity:
-    def __init__(self, name, animation_list=None, health=10, is_occupied_field = True, type = None):
+    def __init__(self, name, deckInv, animation_list=None, health=10, is_occupied_field = True, type = None):
```

Comment:
what is deckInv?

Reply:
It is the dict that used as input params for function self.deck.read_conf() in line 25 

Your initial code was to construct the Player first, then called read config function to load the deck. I move the method called into the construction function of Entity, so that we don't have to directly called it every time after creating the Entity.

Reply:
Oh, so it doesn't have anything to do with inventory right?  I tought Inv was something inventory

Reply:
I use the word deckInv because i thought your read config method is to load the default deck into player's inventory first. IMO it is the parameter of the preset deck that you want to load into user inventory (but I thought your code just add it directly to the deck right?)

Reply:
yeah, directly to the deck

Reply:
So what do you think about the params name? Is it ok to use deckInv or do u want to change it?